pkg/issuer/ca/util_test.go

```diff
@@ -33,8 +33,9 @@ type caFixture struct {
 	CA *CA
 	*test.Builder
 
-	Issuer      v1alpha1.GenericIssuer
-	Certificate *v1alpha1.Certificate
+	Issuer             v1alpha1.GenericIssuer
+	Certificate        *v1alpha1.Certificate
+	CertificateRequest *v1alpha1.CertificateRequest
```

Comment:
(not a comment to address here): I want these helpers to go away 😅

Reply:
:grimacing: